.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug Report
+description: Report a problem and help us fix it.
+labels: ["kind/bug"]
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: In which component did the issue occur?
+      options:
+        - Not sure*
+        - Operate*
+        - Optimize*
+        - Tasklist*
+        - Identity*
+        - Zeebe*
+        - C8Run*
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
```

Comment:
Can we change this to "Description"? A bit more consistent with the other labels (that also don't tell you what to do)